.travis.yml

```
@@ -8,4 +8,5 @@ install:
 - make setup
 
 script:
-- make lint
\ No newline at end of file
+- make lint
+- sphinx-build -n -W -q -b html -d ./docs/_build/doctrees ./docs ./docs/_build/html
```

```suggestion
- sphinx-build -n -W -E -q -b html -d ./docs/_build/doctrees ./docs ./docs/_build/html
```
`-E` flag converts warnings into strict errors